styles.css

```diff
@@ -452,3 +491,358 @@ div.buttoncontact button {
   color: var(--mark-text2);
   margin-bottom: 1em;
 }
+
+@media screen and (min-width: 704px) { /* ********* tablet version ********** */
+
+  button.menubutton { /* ****** top bar ******** */
+    display: none;
+  }
+
+  div.topbar {
+    justify-content: center;
+  }
+
+  div.navbar { /* ********* nav bar ********** */
+    display: flex;
+    width: 100%;
+    margin-top: 3em;
+    justify-content: center;
+  }
+
+  div.navbar ul {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    margin: 1em 0;
+    padding: 0;
+    width: 80%;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  div.navbar ul li a {
+    text-decoration: none;
+    font-family: 'Inconsolata', 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1.25rem;
+    line-height: 1.3125rem;
+    text-transform: capitalize;
+    color: var(--text);
+  }
+
+  div.navbar ul li a:hover {
+    color: var(--mark-text2);
+  }
+
+  div.headline { /* *************** head line ******************* */
+    margin: 9em 0 0 20%;
+  }
```

Comment:
I suggest you put the left side of the margin at 40%
![Captura de pantalla (109)](https://user-images.githubusercontent.com/119135868/228372147-2cca2a97-a3b8-4e48-a2b5-5774436723a0.png)
to make it look like this
![Captura de pantalla (107)](https://user-images.githubusercontent.com/119135868/228372295-bb7e7336-e110-4cfd-b86f-4f70d133052e.png)